backend/src/main/java/gov/cdc/usds/simplereport/api/sms/SmsController.java

```diff
@@ -0,0 +1,32 @@
+package gov.cdc.usds.simplereport.api.sms;
+
+import gov.cdc.usds.simplereport.service.SmsService;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+
+import java.io.IOException;
+
+@ConditionalOnProperty(name="twilio.enabled", havingValue="true")
+@RestController
```

Comment:
Just curious here: since this endpoint is part of the section of SimpleReport used by users with Okta credentials, why make this a rest operation rather than a GraphQL mutation?

Reply:
Not any reason, really, except that it felt weird to make it a GraphQL mutation being that it doesn't change any data – it's strictly a "request" operation

Happy to change it though if you'd prefer, I don't feel strongly either way

Reply:
Yeah, I'm more worried that this makes the security configuration more complicated. Is this controller the only reason we would need to disable CSRF and explicitly list the patient experience paths as bypassing Okta? I think we were fine in #813 with just the `@PreAuthorize` annotation on the controller and didn't need to mess with backend/src/main/java/gov/cdc/usds/simplereport/config/SecurityConfiguration.java

Reply:
No, disabling CSRF is also necessary for `PatientExperienceController` – I hadn't ever scanned the QR code on a mobile device in a deployed environment – so we didn't discover that it didn't work until this integration.

Regardless it's a very small lift to turn this controller into a mutation resolver and add it to the GQL schema – just lmk